sdk/videoanalyzer/video-analyzer-edge/src/index.ts

```diff
@@ -0,0 +1,61 @@
+export * from "./generated/models"; //add omit
```

Comment:
seems like we're maybe overeager in what we export here? Can we slim this down to specific imports that are re-exported?

Reply:
We deleted the 10 models we didn't want to export so now exporting all of them should be fine.